packages/hint/src/lib/config.ts

```diff
@@ -253,17 +252,55 @@ export class Configuration {
         return browserslist(config.browserslist);
     }
 
+    private static groupHintConfigs(configs: ConfigFiles, file: string): ConfigFiles {
+        const basename = path.basename(file) as ConfigFile;
+
+        if (!basename) {
+            throw new Error(`unrecognised config file '${basename}'`);
+        }
+
+        const hintConfig = Configuration.loadConfigFile(file);
+
+        if (basename === ConfigFile.PackageJson) {
+
+            const packageJson = loadJSONFile(file);
+
+            if (packageJson.browserslist) {
+                configs[basename] = packageJson.browserslist;
+            }
+
+            if (hintConfig && hintConfig.browserslist) {
+                configs.hintConfig = hintConfig.browserslist;
```

Comment:
What does this do?

Reply:
It gets the value in the user's hintConfig property declared in the package.json file.

Reply:
Since `loadConfigFile()` gets the `hintConfig` value and `loadJsonFile` gets the contents of the entire package.json.